modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
+
+.Procedure
+Customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. In the *Search* view, select the *Resources* drop down and search "*Console*".
+. Select the *Console `operator.openshift.io/v1`*.
++
+.Searching Console resource
+image::odc_cluster_console.png[]
+. Select *cluster* under the *Name* list.
```

Comment:
Verify that cluster isn't capitalized. In the screenshot, I see all the UI elements are capitalized, so "cluster" here very well may need to be capitalized as well.

Reply:
I'll verify it again.

Reply:
The "cluster" is not capitalized in the UI. Check this:

![image](https://user-images.githubusercontent.com/3268242/152976356-6c56e2dc-f75d-4398-b559-695562770c53.png)

Reply:
Principle of location before action: Suggest: Under the *Name* list, select...
The user needs to know where the field is before he/she takes action on it.